File: python/python_q1639/distractor2.py
import re

doc = 'Mail me at example@example.com\nor example (at) example.co.com'
count = 0
for _ in range(1000000):
    matches = re.finditer(r'.*\b(\w+(@|\s*\(?at\)?\s*)\w+(\.\w+)+)\b.*', doc)
    count += sum(1 for _ in matches)

print(count)

File: cpp/cpp_q1859/distractor2.cpp
#include <iostream>

int main() {
    std::cout << __FILE__ << std::endl;

    int[] array = { 1, 2, 3 };
    int sum = 0;
    for (auto i : array) {
        sum += i;
    }

    std::cout << sum << std::endl;
}